jobs/pending_payments.ts

```diff
@@ -16,6 +16,7 @@ exports.attemptPendingPayments = async (bot: Telegraf<MainContext>): Promise<voi
   });
   for (const pending of pendingPayments) {
     const order = await Order.findOne({ _id: pending.order_id });
+    if (order == null) throw "order object is null";
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program